src/components/Sidebar/Sidebar.tsx

```diff
@@ -1,130 +1,169 @@
-      plant.attributes.image = plantImage
-      return <Card plant={plant.attributes} modal={modal} setModal={setModal} key={plant.attributes.plant_id} />
-    })
-  } else {
-    cards = <p className="loading">Hmmm... there are no plants in our nursery matching your search.</p>
-  }
-
-  const handleSearchChange = (event: ChangeEvent<HTMLInputElement>) => {
-    setSearchTerm(event.target.value);
-  };
-
-  const handleKeyPress = (event: React.KeyboardEvent) => {
-    if (event.key === 'Enter') {
-      searchForPlants();
+    const [plantList, setPlantList] = useState<PlantData[]>([])
+    const [decorList, setDecorList] = useState<DecorData[]>([])
+    // eslint-disable-next-line
+    const [apiError, setApiError] = useState<string>('')
+    const [searchTerm, setSearchTerm] = useState<string>('');
+    const [loadingPlants, setLoadingPlants] = useState<boolean>(true)
+    const [activeTab, setActiveTab] = useState<string>('plants')
+
+    const getDecorList = () => {
+        //this is a placeholder for what will become the api call for the decor list
+        setDecorList(decorData.data)
     }
-  };
-
-  const searchForPlants = () => {
-    setLoadingPlants(true)
-    if (searchTerm.length) {
-      searchPlants(searchTerm.toLowerCase())
-        .then(data => setPlantList(data.data))
-        .then(() => setLoadingPlants(false))
-        .catch((err) => {
-          handleApiError(err)
+
+    useEffect(() => {
+        getPlantList()
+            .then(data => setPlantList(data.data))
+            .then(() => setLoadingPlants(false))
+            .catch((err) => {
+                handleApiError(err)
+            })
+        getDecorList()
+    }, [])
+
+    const handleApiError = (error: string) => {
+        setApiError(error)
+    }
+
+    let cards;
+
+
+    if (!plantList || !decorList) {
+        cards = <p className="loading">It looks like our nursery is not operating correctly, please try again later</p>
+    }
+    else if (plantList.length && activeTab === 'plants') {
+        cards = plantList.map((plant: PlantData) => {
+            const plantImage = plant.attributes.image
+                ? plant.attributes.image
+                : `${process.env.PUBLIC_URL}/images/plant-fallback.png`
+            plant.attributes.image = plantImage
+            return <Card plant={plant.attributes} modal={modal} setModal={setModal} key={plant.attributes.plant_id} />
         })
-    } else {
-      getPlantList()
-        .then(data => setPlantList(data.data))
-        .then(() => setLoadingPlants(false))
-        .catch((err) => {
-          handleApiError(err)
+    } else if (decorList.length && activeTab === 'decor') {
+        cards = decorList.map((item: DecorData) => {
+            return <Card plant={item.attributes} modal={modal} setModal={setModal} key={nanoid()} />
         })
+    } else {
+        cards = <p className="loading">Hmmm... there are nothing in our nursery matching your search.</p>
```

Comment:
Can we change this message to what we had it as before?  I think it sounds nicer.

Reply:
Fixed it!